scripts/pages/index.page.js

```diff
@@ -1,2 +1,46 @@
 
+import Dropdown from 'scripts/components/dropdown.component';
+
 import 'styles/homepage.scss';
+
+const defaults = {
+  commodity: 'Soy',
+  country: 'Brazil'
+};
+
+const _setMap = function() {
+  const imagesItem = document.querySelectorAll('.map-gallery-item');
+  const commodity = defaults.commodity.toLowerCase();
+  const country = defaults.country.toLowerCase();
+  const imageName = `${commodity}-${country}`;
+
+  imagesItem.forEach((imageItem) => {
+    if (imageItem.getAttribute('data-image-name') === imageName) {
+      imageItem.classList.remove('is-hidden');
+    } else {
+      if (!imageItem.classList.contains('is-hidden')) {
+        imageItem.classList.add('is-hidden');
```

Comment:
You don't toggle the class here, it only displays selected item and hide others ones. I don't think toggle will work here.